scss/main.scss

```diff
@@ -0,0 +1,8 @@
+@charset "UTF-8";
+
+@import "../bower_components/bourbon/app/assets/stylesheets/bourbon";
+@import "../bower_components/neat/app/assets/stylesheets/neat";
+
+.container {
+	@include outer-container;
```

Comment:
I think most projects are using 2 spaces for indents in css files. If you feel strongly about this though, you can comment on it on the CSS styleguide draft: https://docs.google.com/a/gsa.gov/document/d/1t6Rjslq9I20reFF3xfW3FVeV2NWFaDKauDj_LQJC5mc/edit.